scripts/iscan

```diff
@@ -46,6 +46,8 @@ Positional arguments:
 Environment variables:
   S3_BUCKET   S3 bucket to upload scan results to (default: '$DEFAULT_S3_BUCKET').
               Setting S3_BUCKET to an empty string disables S3 upload.
+  EXTRA_OPTS  Extra options for ransomware detection for developers.
+              Example of env var content: "--option1 --option2 some-value"
```

Comment:
[optional] I don't find this information helpful.  _Sapienti sat._
```suggestion
```

Reply:
removed